AlfiyaSiddique/TastyTrails
# Feature forgot password

## What does this PR do?

Sends out an email to the user to reset his password along with a link to do so.
The user can reset his password using that link and can get back to his account
validations in place to check if the email exisits

Fixes #10 



## How should this be tested?




- Click forgot password
-  Add email to send the details
- check your email, reset password and try logging in with new password

## Checklist



### Required
